graph/mutators/comment.js

```diff
@@ -187,11 +187,33 @@ const setCommentStatus = ({loaders: {Comments}}, {id, status}) => {
     });
 };
 
+/**
+ * Adds a tag to a Comment
+ * @param {String} id          identifier of the comment  (uuid)
+ * @param {String} tag     name of the tag
+ */
+const addCommentTag = ({user, loaders: {Comments}}, {id, tag}) => {
+  return CommentsService.addTag(id, tag, user.id)
+    .then(() => CommentsService.findById( id ));
```

Comment:
Hey @gobengo, it should be on https://github.com/coralproject/talk/tree/fix-permalink within a few minutes.